client/my-sites/plans-features-main/index.jsx

```diff
@@ -121,120 +120,72 @@ export class PlansFeaturesMain extends Component {
 			siteId,
 			plansWithScroll,
 			isReskinned,
-			isFAQCondensedExperiment,
 			isPlansInsideStepper,
-			is2023PricingGridVisible,
 			intervalType,
 			planTypeSelectorProps,
-			busyOnUpgradeClick,
 			hidePlansFeatureComparison,
 		} = this.props;
 
-		const plans = this.getPlansForPlanFeatures();
-		const visiblePlans = this.getVisiblePlansForPlanFeatures( plans );
-
-		if ( is2023PricingGridVisible ) {
-			const asyncProps = {
-				basePlansPath,
-				domainName,
-				isInSignup,
-				isLandingPage,
-				isLaunchPage,
-				onUpgradeClick,
-				plans,
+		const asyncProps = {
+			basePlansPath,
+			domainName,
+			isInSignup,
+			isLandingPage,
+			isLaunchPage,
+			onUpgradeClick,
+			plans,
+			flowName,
+			redirectTo,
+			visiblePlans,
+			selectedFeature,
+			selectedPlan,
+			withDiscount,
+			discountEndDate,
+			withScroll: plansWithScroll,
+			popularPlanSpec: getPopularPlanSpec( {
 				flowName,
-				redirectTo,
-				visiblePlans,
-				selectedFeature,
-				selectedPlan,
-				withDiscount,
-				discountEndDate,
-				withScroll: plansWithScroll,
-				popularPlanSpec: getPopularPlanSpec( {
-					flowName,
-					customerType,
-					isJetpack,
-					availablePlans: visiblePlans,
-				} ),
-				siteId,
-				isReskinned,
-				isPlansInsideStepper,
-				intervalType,
-				hidePlansFeatureComparison,
-			};
-			const asyncPlanFeatures2023Grid = (
-				<AsyncLoad
-					require="calypso/my-sites/plan-features-2023-grid"
-					{ ...asyncProps }
-					planTypeSelectorProps={ planTypeSelectorProps }
-				/>
-			);
-
-			return (
-				<div
-					className={ classNames(
-						'plans-features-main__group',
-						'is-wpcom',
-						`is-customer-${ customerType }`,
-						'is-2023-pricing-grid',
-						{
-							'is-scrollable': plansWithScroll,
-						}
-					) }
-					data-e2e-plans="wpcom"
-				>
-					{ asyncPlanFeatures2023Grid }
-				</div>
-			);
-		}
+				customerType,
+				isJetpack,
+				availablePlans: visiblePlans,
+			} ),
+			siteId,
+			isReskinned,
+			isPlansInsideStepper,
+			intervalType,
+			hidePlansFeatureComparison,
+		};
```

Comment:
Our next step should probably be to convert this file to TS. I see that the new pricing grid does not use a lot of these props, and it would be great if we can trim this file further.

Reply:
Totally agree :)